*Creado por:*
    
*Isabel Maniega*

# Ejemplo 1

La estatura promedio de los estudiantes de una universidad privada es de 1.85 metros con una desviación estándar de 0.15 metros. ¿Qué porcentaje de estudiantes son más bajos o tan altos como Margie, que mide 2.05 metros?

*Para resolver este problema lo primero que necesitamos es encontrar nuestro Z-score:*

In [1]:
µ = 1.85
x = 2.05
s = 0.15

In [2]:
Zscore = round((x - µ)/ s, 2)
Zscore

1.33

*Ahora necesitamos usar la tabla de puntuación z para encontrar la proporción por debajo de una puntuación de Z 1.33*

*Para usar la tabla z, comenzamos en la columna más a la izquierda y encontramos los primeros dos dígitos de nuestro puntaje z (en este caso, 1.3), luego encontramos el tercer dígito en la parte superior de la tabla. Donde se cruzan la fila y la columna es nuestra proporción por debajo de este puntaje z.*

<img src="tabla_Zscore.png">

*Esto significa que Margie es más alta que el 90,82% de sus compañeros.*

<img src="85%.png">

# Ejemplo 2

Margie también quiere saber qué porcentaje de estudiantes son más altos que ella. Como el área bajo la curva normal es 1, podemos encontrar esa proporción:

In [3]:
mas_Altos = 1 - 0.9082
mas_Altos

0.09179999999999999

In [4]:
mas_Altos *=100
mas_Altos

9.18

El 9.18% son más altos que Margie

# Ejemplo 3

Anne solo mide 1.87 metros. ¿Qué proporción de compañeros de clase se encuentra entre las alturas de Anne y Margie?

*Ya sabemos que el 90.82% de los estudiantes son más bajos que Margie.Así que primero encontraremos el porcentaje que son más bajos que Anne.*

In [5]:
µ = 1.85
x = 1.87
s = 0.15

In [6]:
Zscore = round((x - µ)/ s, 2)
Zscore

0.13

Si usamos la tabla z vemos que este z-score corresponde a una proporción de 0.5517 o 55.17%. Entonces, para obtener la proporción entre los dos, restamos las dos proporciones entre sí. Esa proporción de personas cuya altura está entre la altura de Anne y Margie es:

In [7]:
proporcion = 90.82 - 55.17
proporcion

35.64999999999999

El resultado es de 35.64% está entre las dos.

# Ejercicio 1

En 2007-2008, la altura promedio de un jugador de baloncesto profesional fue de 2,00 metros 
con una desviación estándar de 0,02 metros. Harrison Barnes es un jugador de baloncesto que mide 2,03 
metros. ¿Qué porcentaje de jugadores son más altos que Barnes?

# Ejercicio 2

Chris Paul mide 1,83 metros. ¿Qué proporción de jugadores de baloncesto se encuentran entre 
las alturas de Paul y Barnes?

# Ejercicio 3

El 92 % de los candidatos obtuvo una puntuación tan buena o peor que la de Steve. Si el puntuación 
promedio fue 55 con una desviación estándar de 6 puntos, ¿cuál fue el puntuación de Steve?

*Creado por:*
    
*Isabel Maniega*